### PR TITLE
Feat(personal settings): internal link buttons

### PR DESCRIPTION
**Pull request by Chrissy**
test

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/account/accountEmails.jsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
 import SettingsPageHeader from '../components/settingsPageHeader';
+import InternalLinkButton from '../../../components/internalLinkButton';
 
 const AccountEmails = () => {
   return (
     <div>
       <SettingsPageHeader title="Emails" />
+      <InternalLinkButton to="/settings/account/notifications" icon="icon-stack">
+        Wanna change how many emails you get? Use the notifications panel.
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/account/accountEmails.jsx]: let's wrap this in our translate helper

File: src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx [hunk 2]
```diff
@@ -35,6 +35,9 @@ export default class AccountNotifications extends AsyncView {
             {accountNotificationFields.map(field => {
               return <FormField key={field.title} field={field} />;
             })}
+            <InternalLinkButton to="/settings/account/emails" icon="icon-mail">
+              Looking to add, remove, or route an email? Use the emails panel.
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: translate

File: src/sentry/static/sentry/app/components/internalLinkButton.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const InternalLinkButtonText = styled('div')`
+  flex-grow: 1;
+`;
+
+const StyledInlineSvg = styled(InlineSvg)`
+  margin-right: 0.75em;
+`;
+
+const InternalLinkButton = styled(({to, children, icon, ...props}) => (
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/internalLinkButton.jsx]: We should start thinking about best practices irt emotion.

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
Looks good, I would say InternalLinkButton doesn't really describe it that well... something that describes the yellow box would be better... (e.g. AlertLink or something)

File: src/sentry/static/sentry/app/components/internalLinkButton.jsx (hunk 3 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/internalLinkButton.jsx]: @billyvg yeah i agree. tbh i wasn't just in love with this style of formatting and would be open minded to other ideas.

**Comment by benvinegar** [comment]
Damn.

**Pull request merged**